src/api.ts

```
@@ -0,0 +1,16 @@
+const API_URL = 'https://jsonplaceholder.typicode.com';
+
+const getItems = async<T>(url: string): Promise<T[]> => {
+  const response = await fetch(API_URL + url);
+
+  if (!response.ok) {
```

It would be better to throw an error in this case and handle it to let user knows that something went wrong